### DISCUSSION TITLE
Cant add new property. Error shown as "Have multiple [NameProperty] attributes

### DISCUSSION THREAD

**Discussion by AaronRiboy** [serenity-is/Serenity#6613] [Help] [answered]
<img width="489" alt="image" src="https://user-images.githubusercontent.com/29535650/207011804-a008a625-2c14-47de-85fc-d9ba0de3b414.png">

Im trying to add new property but i have an error. Sorry im new to this framework.

**Comment by minhhungit** [comment, marked as answer]
the message says very clear, you defined multiple [NameProperty] attribute
remove one, it just accept one [NameProperty]

**Reply by minhhungit** [reply]
normally on your Row.cs

**Reply by AaronRiboy** [reply]
thank you brother.Actually i found the error after i posted on disccusion.I dont know how to delete the discussion.
